README.md

```diff
@@ -1,66 +1,110 @@
 # apko: APK-based OCI image builder
 
-Build OCI images for apk-based distributions declaratively!
+Build and publish [OCI container images](https://opencontainers.org/) built from [APK](https://wiki.alpinelinux.org/wiki/Package_management) packages.
 
-## Why
+apko has the following key features:
+
+ - **Fully reproducible by default.** Run apko twice and you will get exactly the same binary.
+ - **Fast.** apko aims to build images in ms.
+ - **Small.** apko generated images only contain what's needed by the application,
+   in the style of [distroless](https://github.com/GoogleContainerTools/distroless)
+ - **SBOM Support.** apko produces a Software Bill of Materials (SBOM) for images, detailing all the packages inside.
+ - **Services.** apko supports using the s6 supervisor tool to run multiple processes
```

Comment:
```suggestion
 - **Services.** apko supports using the [s6 supervision suite](https://skarnet.org/software/s6) to run multiple processes
```

Reply:
Done (didn't use the GH tool as I'd already pushed a change).